lms/djangoapps/certificates/models.py

```diff
@@ -403,6 +403,7 @@ def _revoke_certificate(self, status, mode=None, grade=None, source=None):
 
         # .. event_implemented_name: CERTIFICATE_REVOKED
         CERTIFICATE_REVOKED.send_event(
+            time=self.modified_date.replace(tzinfo=timezone.utc),
```

Comment:
@robrap I also updated the `send_event` call for the `CERTIFICATE_REVOKED` and `CERTIFICATE_CHANGED` events since I was already in here making changes.